test/users/npadmana/fftw/testFFTW.chpl

```diff
@@ -88,7 +159,10 @@ proc runtest(param ndim : int, fn : string) {
 	destroy_plan(fwd);
 	destroy_plan(rev);
 
-	// Test in-place transforms
+  /* Complex <-> complex in-place transform. 
+
+     This is the same calling sequence as above, with the input/output arrays the same.
+  */
 	fwd = plan_dft(A, A, FFTW_FORWARD, FFTW_ESTIMATE);
 	rev = plan_dft(A, A, FFTW_BACKWARD, FFTW_ESTIMATE);
 	A = goodA;
```

Comment:
It's a trivial observation but I note that the statement "A = goodA;" appears twice in a row